src/table/__tests__/TableWrapper.spec.jsx

```diff
@@ -0,0 +1,86 @@
+import '../../../test-setup';
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import TableWrapper from '../TableWrapper';
+import * as TableBodyWrapper from '../TableBodyWrapper';
+import * as TableHeadWrapper from '../TableHeadWrapper';
+
+describe('<TableWrapper />', () => {
+  const sandbox = sinon.createSandbox();
+  let tableData;
+  let setPageInfo;
+  let constraints;
+  let rowsPerPage;
+
+  const mockContent = () => {
+    sandbox.replace(TableBodyWrapper, 'default', () => <tbody />);
+    sandbox.replace(TableHeadWrapper, 'default', () => <thead />);
+  };
+
+  beforeEach(() => {
+    tableData = {
+      size: { qcy: 200 },
+      rows: [{ qText: '1' }],
+    };
+    setPageInfo = sinon.spy();
+    constraints = {};
+    rowsPerPage = 100;
+  });
+
+  beforeEach(() => {
+    sandbox.verifyAndRestore();
+    sandbox.resetHistory();
+  });
+
+  it('should render table', () => {
+    mockContent();
```

Comment:
@niekvanstaveren Can't seem to run this in `beforeEach`, don't see why? Ideas?

Reply:
You have two beforeEach defined, shouldn't the second one be an afterEach? Not sure if that fixes your issue.....

Reply:
of course...